### PR TITLE
Move from kube-dns to core-dns for local-up-cluster.sh

### PR DESCRIPTION
**Pull request by dims**
Still leave room for folks to run the older kube-dns if they choose to,
but it's about time we move local-up to what we tell everyone to use!

Signed-off-by: Davanum Srinivas <davanum@gmail.com>

/kind cleanup
/priority important-soon

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: hack/local-up-cluster.sh [hunk 1]
```diff
@@ -38,6 +38,8 @@ KUBELET_IMAGE=${KUBELET_IMAGE:-""}
 FAIL_SWAP_ON=${FAIL_SWAP_ON:-"false"}
 # Name of the network plugin, eg: "kubenet"
 NET_PLUGIN=${NET_PLUGIN:-""}
+# Name of the dns addon, eg: "kubedns" or "coredns"
```
Comment (pacoxu) [inline hack/local-up-cluster.sh:41]: Nit: `kube-dns`, not `kubedns`?

Comment (dims) [reply hack/local-up-cluster.sh:41]: good catch. thanks @pacoxu

**Pull request merged**